stock_shipment_schedule_report/__manifest__.py

```diff
@@ -4,7 +4,7 @@
 {
     "name": "Shipment Schedule Report",
     "summary": "",
-    "version": "10.0.1.0.0",
+    "version": "10.0.2.0.0",
```

Comment:
The change made doesn't carry major significance in practical sense. We just happen to define models for this report just because the method we adopt requires them. So from my point of view it doesn't have to be a major bump, but you can decide.